src/notifications/templates/notifications/emails/digest.html

```
@@ -1,35 +1,66 @@
-{% load i18n %}<!DOCTYPE html>
+{% load i18n %}{% load markdown_tags %}<!DOCTYPE html>
 <html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)